enos/enos.py

```diff
@@ -117,6 +119,10 @@ def up(env=None, **kwargs):
     base_inventory = seekpath(env['config']['inventory'])
     inventory = os.path.join(env['resultdir'], 'multinode')
     generate_inventory(env['rsc'], base_inventory, inventory)
+
+    if kwargs['--skip-install']:
```

Comment:
:) This would be too easy :)
Actually the subsequent ansible run installs some dependencies that are mandatory to kolla to run. By exiting here, the configuration of the machine won't fit what is expected as input by the next phase.
The same the env data structure is important for the next phase to run, I'm suspecting that it is not properly populated.
In my opinion, I think having a flag which disable the registry installation + the monitoring stack installation is a better way to go. (see #142)